classes/Course.php

```diff
@@ -1453,8 +1455,7 @@ public function enqueue_course_builder_assets() {
 			)
 		);
 
-		wp_localize_script( 'tutor-course-builder', '_tutorobject', $data );
-		wp_set_script_translations( 'tutor-course-builder', 'tutor', tutor()->path . 'languages/' );
+		add_filter( 'tutor_localize_data', fn( $arr ) => $data );
```

Comment:
Why use Arrow Fn here? We can use the following:
`return add_filter( 'tutor_localize_data', $data );`